src/controllers/controlpickermenu.cpp

```diff
@@ -354,46 +354,53 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
 
     // Library Controls
     QMenu* libraryMenu = addSubmenu(tr("Library"));
-    addPrefixedControl("[Playlist]", "ToggleSelectedSidebarItem",
-                       tr("Expand/Collapse View"),
-                       tr("Expand/collapse the selected view (library, playlist..)"),
+    addPrefixedControl("[Library]", "MoveUp",
+                       tr("Move up"),
+                       tr("Equivalent to pressing the UP key on the keyboard"),
                        m_libraryStr, libraryMenu);
-
-    addPrefixedControl("[Playlist]", "SelectPlaylist",
-                       tr("Switch Next/Previous View"),
-                       tr("Switch to the next or previous view (library, playlist..)"),
+    addPrefixedControl("[Library]", "MoveDown",
+                       tr("Move down"),
+                       tr("Equivalent to pressing the DOWN key on the keyboard"),
+                       m_libraryStr, libraryMenu);
+    addPrefixedControl("[Library]", "MoveVertical",
+                       tr("Move up/down"),
+                       tr("Move vertically in either direction using a knob, as if pressing UP/DOWN keys"),
+                       m_libraryStr, libraryMenu);
+    addPrefixedControl("[Library]", "ScrollUp",
+                       tr("Scroll Up"),
+                       tr("Equivalent to pressing the PAGE UP key on the keyboard"),
                        m_libraryStr, libraryMenu);
-    addPrefixedControl("[Playlist]", "SelectNextPlaylist",
```

Comment:
> But I do not think that a solution that mandatory requires shift is even better. In this case I will stick with the current mapping, Since it is easier to control.

I really don't understand... According to the code in `slotSelectSidebarItem()`, it does exactly the same as pressing the UP/DOWN buttons on the keyboard while the sidebar has focus. It sounds like you're saying that it's doing some kind of automatic item expansion that I didn't see in the code when I looked.

i.e. AFAIK what you wrote should be changed to:

In master:

Right/Left to select the sidebar item
Press the right button on the keyboard to expand the playlists item
Right/Left to choose the playlist (now that is has been expanded)
Up/Down to select the track
LoadA/LoadB to load the track